documentation/src/main/asciidoc/topics/ref_rest_cache_managers.adoc

```diff
@@ -36,6 +36,7 @@ If cross-site replication is not configured, {brandname} returns "local".
 * `site_coordinators_address` is an array of logical addresses of site coordinators.
 * `sites_view` The list of sites that participate in cross-site replication. +
 If cross-site replication is not configured, {brandname} returns an empty list.
+* `rebalancing_enabled` is true if rebalancing is enabled. Fetching this property might fail on the server. In that case the property won't be present in the payload.
```

Comment:
Out of curiosity, why it might fail? This doesn't inspire a lot of confidence :)

Reply:
the method raises exception that can raise by a timeout for example because executes a command

Reply:
Servers other than the cluster coordinator have to connect to the coordinator to fetch the status.
It does retry if the coordinator crashes, so I think the only possibility is that the coordinator doesn't reply and the RPC times out.